sobre/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
You need to put a different title in project. If it is refering to 'Sobre' page, change title to 'Página Sobre'.